ngraph/core/include/openvino/runtime/tensor.hpp

```diff
@@ -0,0 +1,156 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief This is a header file for the OpenVINO Runtime tensor API
+ *
+ * @file openvino/runtime/tensor.hpp
+ */
+#pragma once
+
+#include "openvino/core/coordinate.hpp"
+#include "openvino/core/shape.hpp"
+#include "openvino/core/type/element_type.hpp"
+#include "openvino/runtime/allocator.hpp"
+
+namespace InferenceEngine {
+class Blob;
+}  // namespace InferenceEngine
+
+namespace ov {
+namespace runtime {
+class InferRequest;
+class RemoteContext;
+class VariableState;
+}  // namespace runtime
+
+/**
+ * @brief Tensor API holding host memory
+ *
+ * It can throw exceptions safely for the application, where it is properly handled.
+ */
+class OPENVINO_API Tensor {
```

Comment:
If it is openvino/runtime, I think tensor should be located in the `ov::runtime` namespace

Reply:
see above